### ISSUE TITLE
Bug Report for Mac os Big Sur

### ISSUE THREAD

**Issue by nlintas** [antonreshetov/massCode#132]
## Describe the bug

When opening the app sometimes the window appears with only the background color and nothing happens (the library and folders are not fetched). This happens in around 1/3 of app initialisations for me. 

## To Reproduce

1) Go to the app in the applications folder
2) Click on the app's icon
3) See empty window without anything loaded

## Expected behavior

The window to load my library and folders with code

## Screenshots

<img width="1008" alt="Screenshot 2020-12-17 at 10 52 05" src="https://user-images.githubusercontent.com/45565833/102465133-f1ea6400-4055-11eb-9760-fb9533a7c2f2.png">

## Environment

MassCode Software Version: current latest (1.3.0)
OS version and name: macOS Big Sur 11.1

**Comment by scholtzm** [comment]
macOS Catalina and I have the same issue. At first I thought it was caused by permission not being granted for `read/write` operations in the home folder.

Very first launch: could not open 3 .db files (tags.db and 2 other ones, can't remember) with `ENOENT` error
Now I'm getting this error occasionally:

![image](https://user-images.githubusercontent.com/2640934/108052316-38efd800-704c-11eb-92f2-c277230f2634.png)

**Comment by ktalley-vtech** [comment]
Yes, this appears to be a Mac issue in general.  I experienced this same (random) behavior on Catalina, Big Sur, and now on Mojave (I had to have my MBPro restored due to major Catalina upgrade issues).  Normally I can restart (once or twice) and everything will appear correctly again.  Today, however, using v1.3.0 I simply cannot get my snippets to appear (I've restarted 5-6 times).

**Comment by ktalley-vtech** [comment]
![image](https://user-images.githubusercontent.com/54152105/116264638-1232d980-a740-11eb-8ca3-2d66242689ec.png)

I just realized that there was a menu option for opening Developer Tools for this app, and this error is present -- I'm thinking this is the issue?
